src/components/task/details.cjsx

```diff
@@ -0,0 +1,23 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+Time = require '../time'
+ArbitraryHtml = require '../html'
+
+module.exports = React.createClass
+  render: ->
+    {model} = @props
+
+    if model.description_html
+      detailPopover =
+        <BS.Popover title='Instructions' className='-task-details-popover task-details-popover'>
+          <ArbitraryHtml html={model.description_html}/>
```

Comment:
okay, ill look into that.  i saw that the property was called description_html so I assumed that they would have html